packages/cli/snippets/bulk-write.js

```diff
@@ -0,0 +1,103 @@
+const performBulk = async (z, bulkBundle) => {
+  // Grab the line items, preserving the order
+  const rows = bulkBundle.bulk.forEach(({inputData}) => {
```

Comment:
`.forEach()` returns `undefined`:

```
> a = [1, 2, 3]
[ 1, 2, 3 ]
> a.forEach((x) => x + 1)
undefined
```

Should we use `.map()` instead?

Reply:
Updated